modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/paths/APIBuilder.path

```diff
@@ -95,6 +95,11 @@
 	<td>//div[@class='dropdown-menu show']//button[contains(text(),'Edit')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>ENTER_NAME</td>
```

Comment:
We don't need to add a new path if we already have one to use. Could you help check what we are use during creating and see if it works?

Reply:
we don't have the same path, the action is in edit application's schema page